hello-server.js

```diff
@@ -1,14 +1,14 @@
-const http = require('http');
-const port = 3001;
-const server = http.createServer();
+const http = require('http')
+const port = 3001
+const server = http.createServer()
 
 server.on('request', (request, response) => {
-    console.log(`URL: ${request.url}`);
-    response.end('Hello, server!');
-});
+  console.log(`URL: ${request.url}`)
+  response.end('Hello, server!')
+})
 
 server.listen(port, (error) => {
-    if (error) return console.log(`Error: ${error}`);
+  if (error) return console.log(`Error: ${error}`)
 
-    console.log(`Server is listening on port ${port}`);
-});
\ No newline at end of file
+  console.log(`Server is listening on port ${port}`)
+})
```

Comment:
Largely altered inorder to have it conform to standard js format

Reply:
Using template string is a great way to make code more readable.